app/main.py

```diff
@@ -2,0 +1,29 @@
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = False
+        Animal.alive.append(self)
```

Comment:
Appending `self` to the `Animal.alive` list is correct for tracking living animals. Ensure that this logic aligns with the task requirements for managing animal states.